ddR/NAMESPACE

```diff
@@ -40,7 +39,7 @@ export(is.dlist)
 export(is.dobject)
 export(is.sparse_darray)
 export(nparts)
-export(parallel)
+export(parallel.ddR)
```

Comment:
Let's see how the options look together:

1. `library(Spark.ddR); useBackend(Spark.ddR, master = "local", ...)` Current PR
1. `useBackend("spark", master = "local", ...)`
2. `use(ddR$spark, master = "local", ...)`

The 2nd option is appealing, along with the functionality @lawremi mentions. It seems like a clear and concise option.